### ISSUE TITLE
`SearchDirection` enum being submitted to API as `int`

### ISSUE THREAD

**Issue by hognevevle** [notion-dotnet/notion-sdk-net#133]
**Describe the bug**
```
 Notion.Client.NotionApiException: body failed validation: body.sort.direction should be `"ascending"` or `"descending"`, instead was `1`.
         at Notion.Client.RestClient.SendAsync(String requestUri, HttpMethod httpMethod, IDictionary`2 queryParams, IDictionary`2 headers, Action`1 attachContent, CancellationToken cancellationToken)
         at Notion.Client.RestClient.PostAsync[T](String uri, Object body, IDictionary`2 queryParams, IDictionary`2 headers, JsonSerializerSettings serializerSettings, CancellationToken cancellationToken)
         at Notion.Client.SearchClient.SearchAsync(SearchParameters parameters)
```

**To Reproduce**
![CleanShot 2021-09-28 at 13 38 18](https://user-images.githubusercontent.com/2721188/135079757-940eb7f5-e428-4dde-b282-84046123bdd6.png)

**Comment by hognevevle** [comment]
Same applies for SearchFilter.Value

```
 Notion.Client.NotionApiException: body failed validation: body.filter.value should be `"page"` or `"database"`, instead was `1`.
         at Notion.Client.RestClient.SendAsync(String requestUri, HttpMethod httpMethod, IDictionary`2 queryParams, IDictionary`2 headers, Action`1 attachContent, CancellationToken cancellationToken)
         at Notion.Client.RestClient.PostAsync[T](String uri, Object body, IDictionary`2 queryParams, IDictionary`2 headers, JsonSerializerSettings serializerSettings, CancellationToken cancellationToken)
         at Notion.Client.SearchClient.SearchAsync(SearchParameters parameters)
```

**Comment by KoditkarVedant** [comment]
@hognevevle Thank you for your contribution. I've merged your PR and it is released as part of 1.4.2 release.
